misc/test_site/mac_addrs/9a02571e8f02/module_config.json

```diff
@@ -7,14 +10,24 @@
     "name": "*** Name ***",
     "serial": "*** Serial ***",
     "type": "*** Type ***",
-    "version": "*** Version ***"
+    "firmware_version": "*** Firmware Version ***"
   },
+  "device_manuals_url": "*** Device Manuals URL ***",
   "modules": {
-    "switch": {
+    "bacnet": {
+      "enabled": true
+    },
+    "brute": {
+      "enabled": true
+    },
+    "mudgee": {
       "enabled": true
     },
-    "foobar": {
```

Comment:
why this change?

Reply:
I got carried away and copied the same module_config.json from faux1 to faux2 ... 
Does the foobar thingy have any special meaning for you?

Reply:
I've reverted it to the same content of the master branch.